conda/recipes/libcudf/recipe.yaml

```diff
@@ -0,0 +1,233 @@
+      license: ${{ load_from_file("python/libcudf/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libcudf/pyproject.toml").project.description }}
+
+  - package:
+      name: libcudf_kafka
+      version: ${{ version }}
+    build:
+      script:
+        - cmake --install cpp/libcudf_kafka/build
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      host:
+        - librdkafka ${{ librdkafka_version }}
+        - ${{ pin_subpackage('libcudf', exact=True) }}
+      run:
+        - librdkafka ${{ librdkafka_version }}
+        - ${{ pin_subpackage('libcudf', exact=True) }}
+      ignore_run_exports:
+        by_name:
+          - cuda-version
+    tests:
+      - script:
+        - test -f $PREFIX/lib/libcudf_kafka.so
+    about:
+      homepage: https://rapids.ai/
+      license: Apache-2.0
+      summary: libcudf_kafka library
+
+  - package:
+      name: libcudf-example
+      version: ${{ version }}
+    build:
+      script:
+        - ./cpp/examples/build.sh --install
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ compiler('c') }}
+        - ${{ compiler('cxx') }}
+        - ${{ compiler('cuda') }} =${{ cuda_version }}
+        - cuda-version =${{ cuda_version }}
+        - ninja
+        - ${{ stdlib("c") }}
+      host:
+        - ${{ pin_subpackage('libcudf', exact=True) }}
+        - if: cuda_major == "11"
+          then:
+            - cuda-nvtx =${{ cuda_version }}
+          else:
+            - cuda-nvtx-dev
+        - cuda-version =${{ cuda_version }}
+      run:
+        - ${{ pin_subpackage('libcudf', exact=True) }}
+        - ${{ pin_compatible('cuda-version', upper_bound='x', lower_bound='x') }}
+        - if: cuda_major != "11"
+          then:
+            - cuda-nvtx
+      ignore_run_exports:
+        from_package:
+          - if: cuda_major != "11"
+            then:
+              - cuda-nvtx
+        by_name:
+          - cuda-version
+    about:
+      homepage: ${{ load_from_file("python/libcudf/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libcudf/pyproject.toml").project.license.text }}
+      summary: libcudf example executables
+
+  - package:
+      name: libcudf-tests
+      version: ${{ version }}
+    build:
+      script:
+        - cmake --install cpp/build --component testing
+        - cmake --install cpp/libcudf_kafka/build --component testing
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      host:
+        - ${{ pin_subpackage('libcudf', exact=True) }}
+        - ${{ pin_subpackage('libcudf_kafka', exact=True) }}
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then:
+            - libcurand ${{ cuda11_libcurand_run_version }}
+          else:
```

Comment:
Is this empty `else:` correct? Maybe we need `libcurand-dev` here?

Reply:
fixed, thanks